cli/src/main/scala/com/codacy/analysis/cli/command/analyse/AnalyseExecutor.scala

```diff
@@ -155,13 +180,31 @@ class AnalyseExecutor(toolInput: Option[String],
 
 object AnalyseExecutor {
 
-  final case class ExecutorResult(toolName: String, files: Set[Path], analysisResults: Try[Set[ToolResult]])
+  final case class ExecutorResult(toolName: String, files: Set[Path], analysisResults: Try[Set[Result]])
+
+  def languages(localConfiguration: Either[String, CodacyConfigurationFile],
```

Comment:
Why do you need this method? Seems like a strait invocation. If you want to optimise I would do it inside.